doc/source/BuildingRunningTesting/BuildRunLandDA.rst

```diff
@@ -24,14 +26,14 @@ Create a directory for the Land DA experiment (``$LANDDAROOT``):
    cd /path/to/landda
    export LANDDAROOT=`pwd`
```

Comment:
I updated the wording a bit because I do want users to know that they can save the directory as an environment variable to help them navigate from one directory to another without typing out the whole path. I'm hoping it's clear now that this is an optional step ("Optionally, users can...) and that `$LANDDAROOT` is an example variable name. That said, the tutorial I made references `$LANDDAROOT` as an option, so I don't think we should delete this section entirely.